sourcegraph/values.yaml

```diff
@@ -8,6 +8,7 @@ sourcegraph:
   imagePullSecrets: []
   labels: {}
   revisionHistoryLimit: 10
+  localDevMode: false # When true, remove all resource stanzas, allowing the scheduler to best-fit pods. Intended for local development with limited resources.
```

Comment:
I like it! We could achieve the same thing by providing an override file that sets all the resources to empty, but this config setting will let us build in other local-specific features.